core/src/main.cpp

```diff
@@ -819,6 +819,10 @@ auto load_oidc_configuration(const json& _config, json& _oi_config, json& _endpo
 		logging::trace("Invalid OIDC configuration, ignoring. Reason: {}", e.what());
 		return false;
 	}
+	catch (const boost::beast::system_error& e) {
+		logging::trace("Unable to establish connection to OpenID Provider. Reason: [{}]", e.what());
```

Comment:
let's go `debug` or even `info` here...    feels useful to an admin at all times to see this message.

Reply:
Agreed. Let's do it.

Reply:
Done.